python/tvm/autotvm/task/dispatcher.py

```diff
@@ -460,7 +460,15 @@ def _query_inside(self, target, workload):
             self.update(target, workload, cfg)
             return cfg
         key = (str(target), workload)
-        return self._global_cfg_dict[key]
+        if key not in self._global_cfg_dict:
+            msg = "Config for target=%s, workload=%s is missing in ApplyGraphBest context. " \
+                  "A fallback configuration is used, which may bring great performance " \
+                  "regression." % (target, workload)
+            logger.warning(msg)
+            cfg = FallbackConfigEntity()
```

Comment:
We can store this config in `self._global_cfg_dict` as we do in https://github.com/dmlc/tvm/blob/271ecb31a01e2b05d87e0c759bec42032d2d10d0/python/tvm/autotvm/task/dispatcher.py#L367-L369. So all latter changes and querys will be directed to this config.

Then we don't need the changes in the `topi/python/topi/x86/nn.py`